SupyML/plugin.py

```diff
@@ -180,11 +200,24 @@ def _processNode(self, node, variables, nested):
 
         if node.nodeName=='echo':
             # special handling for echo: utils echo does not like empty strings, so we reimplement
-            value = arguments
+            value = self._unescape(arguments)
         elif node.nodeName=='raise':
             # raise an exception within SupyML context
             value = ''
-            self.errors.append(arguments)
+            self.errors.append(self._unescape(arguments))
+        elif node.nodeName=='quot':
+            # quotes the result of the nested expression to be treated as a single token
+            # until a defined number of levels up
+            level = 1
+            if 'l' in node.attributes.keys():
+                try:
+                    level=int(node.attributes['l'].value)
+                except:
+                    pass
```

Comment:
this could make debugging quite hard; why not raise an error instead?

Reply:
quite true. especially since we have such nice error handling now. I shall change that,